zipline/gens/tradesimulation.py

```diff
@@ -114,6 +109,25 @@ def transform(self):
             '2002-01-02')]
         first_trading_day_idx = all_trading_days.searchsorted(trading_days[0])
 
+        if algo.benchmark_sid is None:
+            # get benchmark info from tradingenvironment
+            benchmark_series = env.benchmark_returns[
+                trading_days[0]:trading_days[-1]]
+        else:
+            # calculate benchmark. start one day before the first trading day,
+            # so we can get a return value for the first day.
+            benchmark_sid = algo.benchmark_sid
+            benchmark_series = data_portal.get_history_window(
+                [benchmark_sid],
+                trading_days[-1],
+                bar_count=len(trading_days) + 1,
```

Comment:
I can't recall, is this a change to current behavior. i.e. do we start with 0 returns on the first day, use open to close for first return, or lookback one day like we are doing here.